doc/thing/samples/basic-samples/toggle.rst

```diff
@@ -0,0 +1,53 @@
+toggle
+======
+
+Overview
+--------
+
+The toggle example uses an on-board LED to toggle its state via cloud
+request.
```

Comment:
Please, describe which LED is controlled via cloud request.

Reply:
Ok.